### DISCUSSION TITLE
FCM ( Custom Notification class not working)

### DISCUSSION THREAD

**Discussion by AndroXStudiO** [Sketchware-Pro/Sketchware-Pro#657] [General]
i have a custom class for FCM Notification. 
as MyFirebaseMessagingService.  I have added it in manifest also...but still after run app...this class not working..default notification coming and showing fine with default. 

for that case i also tried with replacing  default service from manifest using Command block.But again notification coming with default 😓.
Please help me

**Comment by AndroXStudiO** [comment]
Any solution?

**Reply by AndroXStudiO** [reply]
@hasrat-ali @JavkhlanK  @hosni-fraj  @khaled-0 
Help me..if any has solution

**Reply by j-px** [reply]
This isn't for general Android development questions, and even if you do, don't be annoying and ping people.

**Reply by j-px** [reply]
I haven't used FCM before and have no idea, so I didn't answer
